tools/nxfuse/src/tinyara_services.c

```diff
@@ -392,7 +392,7 @@ void *smartfs_vmount(const char *datasource, const char *mount_point, uint32_t e
 		return NULL;
 	}
 
-	/* Setup the SmartFS */
+	/* Setup the jSmartFs */
```

Comment:
For changes in line 395, 464 and 501
Though we are using nxfuse to mount our jsmartfs Filesystem, Nxfuse was designed as a tool to handle image creation and mounting of smartfs in general.
Hence, this remains a gray area for change which will be confirmed later based on everybody's opinion.

I think we should definitely change the name of the fs for the mount command but I am not sure whether the function descriptions should be changed or not.